docs/AusTrakka CLI/CLI-sequence-upload.md

```diff
@@ -75,6 +38,18 @@ Having created a `files.csv`, you can upload the sequence files listed in your C
 austrakka seq add fastq-ill-pe --csv files.csv
 ```
 
+This assumes that the `Seq_ID` values map to existing samples. If these are new samples, you can create them by
+appending the `--create`, `--owner-org` and `--shared-projects` to the `seq add` command:
```

Comment:
Does not require --shared-projects, just --create and -oo/--owner-org. Also small grammatical typo ("the" requires "options" or similar later in the sentence, or else drop "the").